responsibleai/tests/test_serialization_utilities.py

```diff
@@ -28,3 +29,13 @@ def test_unknown(self):
         c = complex(1, 2)
         result = serialize_json_safe([c, 42])
         assert result == [c, 42]
+
+    def test_strings_with_special_chars(self):
```

Comment:
Maybe you could add a test that roundtrips this dictionary?
I think your test covers the scenario, but if someone had to update the JSON string below they could do it in a way that roundtripping technically doesn't work.
Like this:
dict -> str (or file)
str (or file) -> dict
assert dict1 == dict2

Reply:
done!